$Id: fix_invalid_sql.patch 379 2012-09-20 11:45:26Z tg $

Fix invalid SQL: Debian #615983
| A database error has occurred
| Query: INSERT IGNORE INTO page_restrictions (pr_page,pr_type,pr_level,pr_cascade,pr_user,pr_expiry) SELECT 793,pr_type,pr_level,pr_cascade,pr_user,pr_expiry FROM page_restrictions WHERE pr_page = '216'
| Function: Title::moveTo
| Error: 1 ERROR: syntax error at or near "IGNORE"
| LINE 1: INSERT /* Title::moveTo Jgeese */ IGNORE INTO page_restricti...
| ^
Reference: https://evolvis.org/tracker/t_follow.php/1377

--- a/includes/Title.php
+++ b/includes/Title.php
@@ -3524,8 +3524,7 @@ class Title {
 					'pr_expiry'  => 'pr_expiry'
 				),
 				array( 'pr_page' => $pageid ),
-				__METHOD__,
-				array( 'IGNORE' )
+				__METHOD__
 			);
 			# Update the protection log
 			$log = new LogPage( 'protect' );
